pkg/operator/ceph/cluster/version.go

```diff
@@ -230,7 +230,7 @@ func (c *cluster) validateCephVersion(version *cephver.CephVersion) error {
 			if c.Spec.SkipUpgradeChecks {
 				logger.Warning("ceph is not healthy but SkipUpgradeChecks is set, forcing upgrade.")
 			} else {
-				return errors.Errorf("ceph status in namespace %s is not healthy, refusing to upgrade. fix the cluster and re-edit the cluster CR to trigger a new orchestation update", c.Namespace)
+				return errors.Errorf("ceph status in namespace %s is not healthy, refusing to upgrade. Either fix the health issue or force an update by setting skipUpgradeChecks to true in the cluster CR", c.Namespace)
```

Comment:
```suggestion
				return errors.Errorf("ceph status in namespace %s is not healthy, refusing to upgrade. To proceed, fix the health issue or force an unsafe update by setting skipUpgradeChecks to true in the CephCluster resource", c.Namespace)
```

Reply:
is it a request change or not? It's confusing since you approved :)